### ISSUE TITLE
ERROR with Event Loop

### ISSUE THREAD

**Issue by warjohn** [Marker-Inc-Korea/AutoRAG#609]
Hello. I wrote a small RAG system for ma project and I decided to push it on localhost

```
host = "localhost"
runner = Runner.from_yaml('RAG_system/data_original/test_1/pipline.yaml', project_dir = "RAG_system/data_original/test_1")
runner.run_api_server(host)

```
My code for posting 

```
import requests 
url = "http://localhost:8000/run"
data = {
    "query": "query",
    "result_column": "Answer"
}
import json```
payload = json.dumps(data)
headers = {
    'accept': "application/json",
    'Content-Type': "application/json"
    }
response = requests.request("POST", url, data=payload, headers = headers)
print(response.text)
```

And every time I get this error  
### RuntimeError: This event loop is already running
How do I solve it?

**Comment by vkehfdl1** [comment]
@warjohn Hi, did you run this code in jupyter?

Try this

```python
import nest_asyncio
nest_asyncio.apply()
```

**Comment by warjohn** [comment]
it doen't work on linux, idk

**Comment by vkehfdl1** [comment]
@warjohn 
I am investigating this problem. 
Thanks for reporting the issue.

**Comment by vkehfdl1** [comment]
@warjohn 
I made PR to resolve this error.
Thanks for reporting a but for me again.
This fix will be update on AutoRAG v0.2.14

**Comment by hongsw** [comment]
> [@warjohn](https://github.com/warjohn) Hi, did you run this code in jupyter?
> 
> Try this
> 
> import nest_asyncio
> nest_asyncio.apply()

Thanks ![Image](https://github.com/user-attachments/assets/4147137f-01ec-4c8a-9290-4845bb4f40ac)
